### ISSUE TITLE
Integration test of room metadata

### ISSUE THREAD

**Issue by cryptix** [ssb-ngi-pointer/go-ssb-room#219]
> Something like 
> - go-ssb-room creates invite
> - Node.js client accepts an invite from the Go server using ssb-http-invite-client
> - Node.js client connects to the go-ssb-room
> - isRoom happens in the background and Node.js client saves data in conn.json
> - We check what was written to conn.json
>
> https://github.com/ssb-ngi-pointer/go-ssb-room/pull/218#pullrequestreview-660801159

Personally, I want to be sure we don't regress on this schema/format.

**Comment by staltz** [comment]
> Personally, I want to be sure we don't regress on this schema/format.

Yeah, and I was thinking where to specify this format. Probably best will be on the rooms2 spec but for `room.metadata()` once we implement that.
